client/my-sites/plans-grid/components/billing-timeframe.tsx

```diff
@@ -81,82 +81,85 @@ function usePerMonthDescription( { planSlug }: { planSlug: PlanSlug } ) {
 	if ( introOffer?.intervalCount && introOffer.intervalUnit ) {
 		if ( discountedPriceFullTermText ) {
 			if ( isMonthlyPlan ) {
-				return translate(
-					'per month, for your first %(introOfferIntervalCount)s %(introOfferIntervalPeriod)s,{{br/}}' +
-						'then, %(rawPrice)s for the first month, Excl. Taxes',
-					'per month, for your first %(introOfferIntervalCount)s %(introOfferIntervalPeriod)ss,{{br/}}' +
-						'then, %(rawPrice)s for the first month, Excl. Taxes',
-					{
-						count: introOffer.intervalCount,
-						args: {
-							rawPrice: discountedPriceFullTermText,
-							introOfferIntervalCount: introOffer.intervalCount,
-							introOfferIntervalPeriod: introOffer.intervalUnit,
-						},
-						components: { br: <br /> },
-						comment: 'Excl. Taxes is short for excluding taxes',
-					}
-				);
+				if ( 'month' === introOffer.intervalUnit ) {
+					return translate(
+						'per month for your first %(introOfferIntervalCount)s month,{{br/}}' +
```

Comment:
`Non-blocking / Nit:`

I'm unsure how much this matters, but you had outlined a requirement whereby one of the minor changes we needed to make was `doesn’t force the string to wrap after 3 months` peapX7-3fa-p2#comment-4783

Wanted to point this out just in case, but feel free to ignore if the situation has changed

Reply:
Thanks. It looks like the wrapping will stay after all. The latest copy can be found here: peapX7-3fa-p2#comment-4841